web/gui/dashboard_info.js

```diff
@@ -1436,7 +1455,7 @@ netdataDashboard.context = {
     },
 
     'system.swapcalls': {
-        info: 'Monitor calls to functions <code>swap_readpage</code> and <code>swap_writepage</code>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows swap access per <a href="#menu_apps_submenu_swap">application</a>.'
+        info: 'Number of calls to <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#swap" target="_blank">functions</a> used to manipulate swap data. Netdata shows swap metrics per <a href="#ebpf_apps_swap_read">application</a> and <a href="#ebpf_services_swap_read">cgroup (systemd Services)</a> if <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">apps</a> or <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#integration-with-cgroupsplugin" target="_blank">cgroup (systemd Services)</a> plugins are enabled.' + ebpfChartProvides + '<div id="ebpf_global_swap"></div>'
```

Comment:
Does it make sense to have it (for instance) as a family description and not for **every** chart? Or is it supposed to be like that?

I see we keep adding the same text again and again and again

<img width="1410" alt="Screenshot 2022-04-08 at 12 07 11" src="https://user-images.githubusercontent.com/22274335/162403845-b1ccdf94-d378-4feb-a455-607b3239453a.png">

Reply:
I talked this with documentation team and our product manager, and they did not see problems, because some charts have different links, but they have more or less the same text. This applies for the charts that you are showing.